### PR TITLE
Updates based off PR review.

### PR DESCRIPTION
**Pull request by sarahboyce**

### CODE REVIEW

File: indymeet/templates/registration/login.html [hunk 1]
```diff
@@ -23,13 +23,7 @@
...
             </div>
         {% endif %}
-        {% if form.errors and not form.non_field_errors %}
-          <div class="alert alert-danger">
-          {% for key, error in form.errors.items %}
-          <p> {{ key }}:{{ error }}</p>
-          {% endfor %}
-          </div>
-          {% endif %}
+        {% include 'includes/no_field_errors.html' %}
```
Comment (tim-schilling) [inline indymeet/templates/registration/login.html:26]: Wait, isn't this only rendering the `non_field_errors` now though?

File: indymeet/settings/production.py [hunk 2]
```diff
@@ -55,10 +55,7 @@
     STATIC_URL = f"https://{AZURE_CUSTOM_DOMAIN}/{AZURE_STATIC_CONTAINER}/"
     MEDIA_URL = f"https://{AZURE_CUSTOM_DOMAIN}/{AZURE_MEDIA_CONTAINER}/"
 
-    SENTRY_DNS = os.environ.get(
-        "SENTRY_DNS",
-        "https://302dde9a33fabc5af106ccf19ae23680@o4506747118157824.ingest.sentry.io/4506747129626624"  # Staging
-    )
+    SENTRY_DNS = os.environ.get("SENTRY_DNS")
```
Comment (tim-schilling) [inline indymeet/settings/production.py:58]: I created a new DSN for staging and added it as an environment variable to our azure staging environment. When you merge this, I'll delete the DSN we had previously.

File: indymeet/templates/registration/login.html (hunk 1 above)
Comment (sarahboyce) [reply indymeet/templates/registration/login.html:26]: Yes but the other errors are rendered under the fields, I don't think we need to put them twice.

Comment (tim-schilling) [reply indymeet/templates/registration/login.html:26]: Ah, that makes sense. Good catch

**Review by tim-schilling — APPROVED** [review]
:rocket:

**Pull request merged**